integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -0,0 +1,143 @@
+# Default values for alluxio.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+image: alluxio/alluxio
+imageTag: 2.1.0-SNAPSHOT
+imagePullPolicy: Always
+
+# Set the tieredstore
+tieredstore:
+  levels:
+# How to use multiple layers
+# level 0 must be MEM, and it's better to point to /dev/shm
+#  - alias: MEM
+#    level: 0
+#    path: /dev/shm
+#    #type: emptyDir
+#    type: hostPath
+#    quota: 10GB
+#    high: 0.95
+#    low: 0.7
+#  - alias: SSD
+#    level: 1
+## The path is inside the container
+#    path: /alluxio-ssd
+## the volume type, it may be hostPath, emptyDir
+#    type: hostPath
+#    quota: 30GB
+#    high: 0.9
+#    low: 0.7
+
+# How to use single layer
+  - alias: MEM
+    mediumtype: MEM,SSD
+    level: 0
+    path: /dev/shm,/alluxio-ssd
+    type: hostPath
+    quota: 10GB,50GB
+    high: 0.95
+    low: 0.7
+
+
+
+user: 0
+group: 0
+fsGroup: 0
+
+domainHostPath: "/tmp/alluxio-domain"
+
+nodeSelector: {}
+
+# properties for all the components
+properties:
+  alluxio.master.journal.type: UFS
+  alluxio.master.journal.folder: /journal
+  # alluxio.user.metrics.collection.enabled: 'true'
+  alluxio.security.stale.channel.purge.interval: 365d
+
+# set for the heap for the docker
+jvmOptions: "-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:MaxRAMFraction=2 "
+
+fuse:
+  image: alluxio/alluxio-fuse
+  imageTag: 2.1.0-SNAPSHOT
+  imagePullPolicy: Always
+  # Customize the MaxDirectMemorySize
+  jvmOptions: " -XX:MaxDirectMemorySize=2g "
+  # shortCircuitPolicy can be local or uuid,
+  # local means the cache directory is in the same mount namespace,
+  # uuid means interact with domain socket
+  shortCircuitPolicy: local
+  # ["fuse", "--fuse-opts=allow_other"]
+  args:
+    - fuse
+    - --fuse-opts=allow_other
+
+master:
+  replicaCount: 1
+  args:
+    - master
+    - -no-format
+# properties for the master components
+  properties:
+    # use ROCKS DB instead of Heap
```

Comment:
I suggest adding keyword `Example` to be more clear:

# Example: use ROCKS DB instead of Heap